lib/components/T3Form/T3FormFieldList/T3FormFieldList.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div>
+    <div
+      v-for="(field, keyItem) in elements"
+      :key="keyItem"
```

Comment:
We shouldn't rely on the iteration index. We can use the more specific keys, like `field.identifier` ?